server/routerlicious/packages/services-core/src/database.ts

```diff
@@ -153,7 +153,7 @@ export interface IDb {
      * Removes a collection or view from the database.
      * The method also removes any indexes associated with the dropped collection.
      */
-    dropCollection(name: string): Promise<boolean>;
```

Comment:
Do we need to add note in breaking.md for this?

Reply:
we haven't released this yet, so I don't think we do